faker/providers/person/ms_MY/__init__.py

```diff
@@ -0,0 +1,5482 @@
+        "{{muhammadName}}{{haji}}{{titleMaleMalay}}{{firstNameMaleMalay}} bin {{titleMaleMalay}}{{firstNameMaleMalay}}",
+        "{{muhammadName}}{{haji}}{{titleMaleMalay}}{{firstNameMaleMalay}} bin {{titleMaleMalay}}{{lastNameMalay}}",
+        # Chinese
+        "{{lastNameChinese}} {{firstNameMaleChinese}}",
+        "{{lastNameChinese}} {{firstNameMaleChinese}}",
+        "{{lastNameChinese}} {{firstNameMaleChinese}}",
+        "{{lastNameChinese}} {{firstNameMaleChinese}}",
+        "{{lastNameChinese}} {{firstNameMaleChinese}}",
+        "{{firstNameMaleChristian}} {{lastNameChinese}} {{firstNameMaleChinese}}",
+        # Indian
+        "{{initialIndian}} {{firstNameMaleIndian}}",
+        "{{initialIndian}} {{lastNameIndian}}",
+        "{{firstNameMaleIndian}} a/l {{firstNameMaleIndian}}",
+        "{{firstNameMaleIndian}} a/l {{firstNameMaleIndian}} {{lastNameIndian}}",
+        "{{firstNameMaleIndian}} {{lastNameIndian}} a/l {{lastNameIndian}}",
+        "{{firstNameMaleIndian}} {{lastNameIndian}} a/l {{firstNameMaleIndian}} {{lastNameIndian}}",
+        "{{firstNameMaleIndian}} {{lastNameIndian}}",
+    )
+
+    femaleNameFormats = (
+        # Malay
+        "{{nurName}}{{hajjah}}{{firstNameFemaleMalay}} {{lastNameMalay}} binti {{titleMaleMalay}}{{firstNameMaleMalay}} {{lastNameMalay}}",
+        "{{nurName}}{{hajjah}}{{firstNameFemaleMalay}} {{lastNameMalay}} binti {{titleMaleMalay}}{{firstNameMaleMalay}}",
+        "{{nurName}}{{hajjah}}{{firstNameFemaleMalay}} {{lastNameMalay}} binti {{titleMaleMalay}}{{lastNameMalay}}",
+        "{{nurName}}{{hajjah}}{{firstNameFemaleMalay}} {{lastNameMalay}}",
+        "{{nurName}}{{hajjah}}{{firstNameFemaleMalay}} binti {{titleMaleMalay}}{{firstNameMaleMalay}} {{lastNameMalay}}",
+        "{{nurName}}{{hajjah}}{{firstNameFemaleMalay}} binti {{titleMaleMalay}}{{firstNameMaleMalay}}",
+        "{{nurName}}{{hajjah}}{{firstNameFemaleMalay}} binti {{titleMaleMalay}}{{lastNameMalay}}",
+        # Chinese
+        "{{lastNameChinese}} {{firstNameFemaleChinese}}",
+        "{{lastNameChinese}} {{firstNameFemaleChinese}}",
+        "{{lastNameChinese}} {{firstNameFemaleChinese}}",
+        "{{lastNameChinese}} {{firstNameFemaleChinese}}",
+        "{{lastNameChinese}} {{firstNameFemaleChinese}}",
+        "{{firstNameFemaleChristian}} {{lastNameChinese}} {{firstNameFemaleChinese}}",
+        # Indian
+        "{{initialIndian}}{{firstNameFemaleIndian}}",
+        "{{initialIndian}}{{lastNameIndian}}",
+        "{{firstNameFemaleIndian}} a/l {{firstNameMaleIndian}}",
+        "{{firstNameFemaleIndian}} a/l {{firstNameMaleIndian}} {{lastNameIndian}}",
+        "{{firstNameFemaleIndian}} {{lastNameIndian}} a/l {{firstNameMaleIndian}}",
+        "{{firstNameFemaleIndian}} {{lastNameIndian}} a/l {{firstNameMaleIndian}} {{lastNameIndian}}",
+        "{{firstNameFemaleIndian}} {{lastNameIndian}}",
+    )
+
+    firstNameMaleMalay_tuple = (
```

Comment:
Can you add sources for this data?

Reply:
Hi @fcurella, they are ported from the php faker https://github.com/fzaninotto/Faker/blob/master/src/Faker/Provider/ms_MY/Person.php

can i just include the github link where it is ported it from?

Reply:
You can follow the instructions [here](https://github.com/joke2k/faker/blob/master/README.rst#tests)

Reply:
Yes, that would work 👍🏼

Reply:
sure, let me add the faker php link as the source. will update at the next version with the tests file.